### PR TITLE
Desktop Version Pull Request

### PR DESCRIPTION
**Pull request by sulaimonaa**
Restructure the HTML file to adapt the desktop view
Added CSS for desktop >= 768px

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -136,85 +149,99 @@ <h4>2015</h4>
...
+                        </li>
+                        <li>
+                            <textarea id="msg" name="user_message" maxlength="500"
+                                required>Write your message here...</textarea>
+                        </li>
+                        <li>
+                            <input type="submit" value="Get in touch">
+                        </li>
+                    </ul>
+                </form>
```
Comment (AbayomiOlaoye) [inline index.html:242]: - [ ] Your page looks amazingly responsive :clap: except that I think there's a slight change that can make the user interface more compelling so much that it enhances user experience across all devices. If you check the mobile design and the desktop design, you will notice the `contact form` is aligned at the center just as the `contact title` and `contact description`. Kindly align the form centrally just as you did in `width > 768px`, and see how interesting it gets :smile: 
- Here's the look at `wdith < 768px`:

image

File: index.html [hunk 2]
```diff
@@ -16,33 +16,47 @@
...
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">Portfolio</a>
+                    </li>
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">About</a>
+                    </li>
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">Contact</a>
+                    </li>
+                </ul>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] [OPTIONAL] To further improve accessibility and user experience via your page navigation, adding internal links that redirect users to appropriate sections of the page (`portfolio`, `about`, and `contact`) will show your commitment to good UI/UX. You can implement this with the use of `id`'s.

_The navigation links are static at the moment, but could be made live if you include correct sectional id's in the `href` attribute._

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @sulaimonaa :wave:, 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 
- Nicely implemented UI :heavy_check_mark: 
- Fairly responsive portfolio page :heavy_check_mark: 
## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @sulaimonaa, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Nice use of media queries ✅
- Good responsive design ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — COMMENTED** [review]
Hi @sulaimonaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great work implementing the desktop version👏👏👏
- [x] 

## Optional suggestions
- [Optional] Kindly add the contributions section in your readme file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**